requirements.txt

```diff
@@ -0,0 +1 @@
+# this file is intentionally left blank
```

Comment:
At the moment, when you run requirements.update, it puts the dependencies for whatever OS you ran it on into requirements.txt rather than into the linux or darwin specific file. I'm not sure if this is a bug or intentional.

So what you need to do when updating is them `mv requirements.txt requirements_darwin.txt` or linux, depending on where you ran the update, then restore the blank file.

This possibly warrants further investigation into whether the requirements.update can directly update the file for the correct OS.